cmd/plural/validation.go

```diff
@@ -49,15 +49,15 @@ func owned(fn func(*cli.Context) error) func(*cli.Context) error {
 	}
 }
 
-func confirmed(fn func(*cli.Context) error, msg string) func(*cli.Context) error {
-	return func(c *cli.Context) error {
-		if ok := confirm(msg); !ok {
-			return nil
-		}
+// func confirmed(fn func(*cli.Context) error, msg string) func(*cli.Context) error {
```

Comment:
is this really unused?  If it is, we can just eliminate it.  I know i used it previously